bloomrun.js

```diff
@@ -35,7 +35,11 @@ BloomRun.prototype.add = function (obj) {
   Object.keys(obj).forEach(function (key) {
     properties.add(key)
   })
-  bucket.data.push(obj)
+
+  bucket.data.push({
+    key: obj,
+    payload: payload || obj
+  })
```

Comment:
Can you use a class here? It would be much faster at scale. Something like https://github.com/mcollina/bloomrun/blob/master/lib/bucket.js.